components/shapediver/parameter/select/SelectImageDropDownComponent.module.css

```diff
@@ -18,5 +24,6 @@
 	:global(.mantine-Input-input) {
 		border: none; /* Hide default border */
 		border-radius: var(--mantine-radius-default);
+		border-color: var(--card-selected-color, var(--input-bd));
```

Comment:
@LemToUp quick question: Where is the --input-bd variable defined?

Reply:
@snabela --input-bd and --input-bd-focus are defined by mantine.

Reply:
@LemToUp many thanks for letting me know!